docs/source/run.rst

```diff
@@ -64,6 +64,10 @@ Commonly Used Arguments
   adjust the name of Gunicorn process as they appear in the process system
   table (which affects tools like ``ps`` and ``top``).
 
+.. Gevent note::
```

Comment:
that's probably better indeed
On Sat, 3 Oct 2015 at 17:39, Berker Peksag <notifications@github.com> wrote:

> In docs/source/run.rst
> <https://github.com/benoitc/gunicorn/pull/1121#discussion_r41088242>:
>
> > @@ -64,6 +64,10 @@ Commonly Used Arguments
> >    adjust the name of Gunicorn process as they appear in the process system
> >    table (which affects tools like ``ps`` and ``top``).
> >
> > +.. Gevent note::
>
> .. note::
>    Foo bar [...]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benoitc/gunicorn/pull/1121/files#r41088242>.
>